doc/tutorial/tutorial.mdk

```diff
@@ -143,16 +161,29 @@ the [source code on github] using these [instructions].
 [source code on github]: http://github.com/FStarLang/FStar
 [instructions]: https://github.com/FStarLang/FStar/blob/master/INSTALL.md
 
-You can edit F\* code using your favorite text editor, but [Emacs],
-[Atom], and [Vim] have extensions that add special support for F\*,
-including syntax highlighting and interactive development. More
-details on [editor support] on the [F\* wiki].
+You can edit F\* code using your favorite text editor, but [Emacs] has an
+extension that add special support for F\*, including syntax highlighting and
+interactive development. More details on [editor support] on the [F\* wiki].
+You can find the code for the exercises and the solutions in the [F\* repository] 
+on Github. However, note that for some exercises, you have to include additional
+libraries. Details on how to do this for Emacs can be found in the wiki.
+
+~KK
+Both Atom and Vim support seems to be either broken or discontinued (or both),
+so it might be misleading to list them as viable options for the audience of this
+tutorial. Also, the code available on the tutorial page and on github differs
+quite a bit (as does the F\* version I guess). In my case, this lead to some
+unexpected errors when copying code from the online-editor to Emacs. It would be

```

Comment:
This was a while ago and I believe the tutorial has been updated since then. The method I used was to clone the FStar Github repo, build fstar.exe and then try to solve the tutorial exercises (copied from the tutorial website) with the bleeding edge version of F*.  This lead to some confusion, because some exercises (on the tutorial website) were written in previous versions of F*. It only dawned on me later that I should have used the versions of the exercises that are in the repo I had cloned as opposed to the ones on the tutorial website.
I see two solutions to the problem. (1) Always keep the tutorial up-to-date and (2) Add a note to in the beginning of the tutorial which version of F* the tutorial reflects. (1) might be a bit unrealistic given the tempo of the F* development. However, (2) would have helped me quite a bit back then.